static/app/views/performance/newTraceDetails/traceDrawer/details/durationComparison.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from '@emotion/styled';
+
+import {Row} from 'sentry/components/events/interfaces/spans/newTraceDetailsSpanDetails';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {getDuration} from 'sentry/utils/formatters';
+
+type Props = {
+  duration: number;
+  title: string;
+  avgDuration?: number;
+  toolTipText?: string;
+};
+
+export const COLORS = {
+  faster: {
+    light: 'green100',
+    normal: 'green300',
+  },
+  slower: {
+    light: 'red100',
+    normal: 'red300',
+  },
+  equal: {
+    light: 'gray100',
+    normal: 'gray300',
+  },
+};
+
+function DurationComparison(props: Props) {
+  if (!props.avgDuration) {
+    return (
+      <Row title={props.title} toolTipText={props.toolTipText}>
```

Comment:
You are coupling the Duration Comparison with the row - this is not good as it means in the future we cannot just take this component and use it anywhere we want. I would remove the row and just render <Duration> with its content, adding the row and tooltip text is small enough where callers can do it.